lib/config-android.h

```diff
@@ -0,0 +1,1092 @@
+/* Manually modified __aarch64__ values. */
```

Comment:
I don't think it's a good idea to modify a file that is in general supposed to be generated. Maybe it's better to generate two and include the correct one based on arch?

Reply:
I've added separate config files for each architecture.